knowledge_repo/_version.py

```diff
@@ -35,7 +35,7 @@
     'Flask-Migrate',  # Database migration utilities
     'sqlalchemy',  # Database abstractions
     'jinja2>=2.7',  # Templating engine
-    'werkzeug',  # Development webserver
+    'werkzeug<1.0',  # Development webserver
```

Comment:
This is required per [this](https://github.com/scoringengine/scoringengine/issues/670) issue.